packages/appsync-emulator-serverless/schema.js

```diff
@@ -137,6 +137,10 @@ const runRequestVTL = (fullPath, graphqlInfo) => {
 };
 
 const runResponseVTL = (fullPath, graphqlInfo, result) => {
+  if (typeof result === 'string') {
+      result = result.replace(/\'/g, '"');
+      result = JSON.parse(result);
+  }
```

Comment:
what is this for?

Reply:
When this method receives the information from the Lambda after the spawn, instead of being an object (like in nodejs inside nodejs) it comes as a JSON string with the data, I only parse it to be rendered by the VTL Render.

Reply:
Can we just put this logic in the python lambda runner ?

Reply:
Solved :) it was simple!